### PR TITLE
Format source code events same as core

### PR DESCRIPTION
**Pull request by afrittoli**
Reformat the source code version control stage page to match what
done for the core stage. No spec changes.

Part of #13

Signed-off-by: Andrea Frittoli <andrea.frittoli@gmail.com>

### CODE REVIEW

File: source-code-version-control.md [hunk 1]
```diff
@@ -10,37 +10,130 @@ description: >
...
+| name  | `String` | The name of the `repository` | `spec`, `community`, `a-repo` | ✅ |
+| owner | `String` | The owner of the `repository` | `cdevents`, `an-org`, `an-user`| ⚪ |
+| url | `URI` | URL to the `repository` | `git://my-git.example/an-org/a-repo` | ✅ |
+| viewUrl | `URI` | URL for humans to view the content of the `repository`  | `https://my-git.example/an-org/a-repo/view`| ⚪ |
+
+### `repository modified`
+
+A Source Code Repository modified some of its attributes, like location, or owner.
+
+- Event Type: __`dev.cdevents.repository.created`__
```
Comment (e-backmark-ericsson) [inline source-code-version-control.md:71]:
```suggestion
- Event Type: __`dev.cdevents.repository.modified`__
```

File: source-code-version-control.md [hunk 2]
```diff
@@ -10,37 +10,130 @@ description: >
...
+| id    | `String` | Uniquely identifies the subject within the source. | `an-org/a-repo`, `an-user/a-repo`, `repo123` | ✅ |
+| source | `URI-Reference` | [source](../spec.md#source) from the context | `my-git.example`| ⚪ |
+| name  | `String` | The name of the `repository` | `spec`, `community`, `a-repo` | ✅ |
+| owner | `String` | The owner of the `repository` | `cdevents`, `an-org`, `an-user`| ⚪ |
+| url | `URI` | URL to the `repository` | `git://my-git.example/an-org/a-repo` | ✅ |
+| viewUrl | `URI` | URL for humans to view the content of the `repository`  | `https://my-git.example/an-org/a-repo/view`| ⚪ |
+
+### `repository deleted`
+
+- Event Type: __`dev.cdevents.repository.created`__
```
Comment (e-backmark-ericsson) [inline source-code-version-control.md:86]:
```suggestion
- Event Type: __`dev.cdevents.repository.deleted`__
```

**Review by e-backmark-ericsson — CHANGES_REQUESTED** [review]
Small typos

File: source-code-version-control.md (hunk 1 above)
Comment (afrittoli) [reply source-code-version-control.md:71]: Nice catch, thank you!

File: source-code-version-control.md (hunk 2 above)
Comment (afrittoli) [reply source-code-version-control.md:86]: Nice catch, thank you!

**Review by e-backmark-ericsson — APPROVED** [review]

**Pull request merged**